### ISSUE TITLE
Multiple subscriptions without names silently fail

### ISSUE THREAD

**Issue by tkurki** [SignalK/signalk-js-client#73]
```
const Client = require('./').Client

const client = new Client({
    hostname: 'localhost',
    port: 3000,
    useTLS: false
})
client.on('connectionInfo', x => console.log(x))
client.connect().then(() =>
    client.subscribe({
        context: 'vessels.self',
        subscribe: [{ path: 'navigation.position' }]
    })
).then(() =>
    client.subscribe({
        context: 'vessels.self',
        subscribe: [{ path: 'navigation.speedOverGround' }]
    })
).then(s => {
    s.unsubscribe()
}).catch(e => console.error(e))
```

The second subscription is never sent to the server, an error message is logged. 

If `name` is required for the API to work correctly why not make it required? Or make the second subscribe fail with a clear error message about what is going on.

The API also relies on the user providing not only names, but unique names. I can't think of a use case for this, am I missing something?

**Comment by fabdrol** [comment]
@tkurki the idea behind the names was that one could have multiple subscriptions for different data, and manage those separate subscriptions. But, despite there being theoretical support for this, I don't think it's possible atm over the single WS connection?

I still think it's a cool feature. Since I'd like to implement this properly, a question re node server: what happens if you send a subscription command to a WS connection that already has an active subscription? Are these combined on the server, or does the second one overwrite the first one?

**Comment by fabdrol** [comment]
This will be fixed by #76 , since named subscriptions will be a thing of the past (and perhaps the future).

**Comment by fabdrol** [comment]
This is no longer relevant, as named subscriptions are a thing of the past for now.
